src/assets/styles/collections/_projects.css

```diff
@@ -129,25 +129,31 @@
 
 @media (width >= 70.875rem) {
     .project .banner {
-        box-shadow:
-            -75rem 0 0 0 var(--fl-fgColor, var(--project-color)),
-            inset 3.5rem 0 0 0 var(--fl-fgColor, var(--project-color)),
-            inset 0 -0.1875rem 0 0 var(--fl-fgColor, transparent);
-        flex-direction: row;
-        height: 39.625rem;
+        display: grid;
+        gap: 0;
```

Comment:
Since we have two grid items in a grid container, it might be cleaner to use gap to create paddings in between the two items. Right now, we have `padding-inline-start: var(--common-inline-padding);` set on banner__content and we can remove that line and change this gap value to create the space between the banner image and the banner content.